Sources/App/Controllers/KeywordController.swift

```diff
@@ -22,11 +22,7 @@ struct KeywordController {
     static func query(on database: Database, keyword: String, page: Int, pageSize: Int) -> EventLoopFuture<Page<Joined3<Package, Repository, Version>>> {
         Joined3<Package, Repository, Version>
             .query(on: database, version: .defaultBranch)
-            .filter(
-                DatabaseQuery.Field.path(Repository.path(for: \.$keywords), schema: Repository.schema),
-                DatabaseQuery.Filter.Method.custom("@>"),
-                DatabaseQuery.Value.bind([keyword])
-            )
+            .filter(Repository.self, \.$keywords, .custom("@>"), [keyword])
```

Comment:
Fluent was missing an overload previously but now we can write this more concisely.